## Prueba #0101

#### Provide the answers in Python code:

1).  En la clase A, las alturas de los estudiantes son:

161.5 <br>
165.9 <br>
168.7 <br>
163.1 <br>
170.8 <br>
167.2 <br>

Calcular la altura media utilizando 'list' y el bucle 'for' de Python:

In [1]:
#Creo una lista con las alturas de la clase A
claseA = [161.5, 165.9, 168.7, 163.1, 170.8, 167.2]
alturaTotal = 0

for altura in claseA:
    alturaTotal += altura

print('Altura media clase A: ',alturaTotal/len(claseA))


Altura media clase A:  166.20000000000002


2).  En la clase B, las alturas de los estudiantes son:

172.5 <br>
162.9 <br>
161.7 <br>
160.1 <br>
161.8 <br>
169.2 <br>

Calcular la altura media utilizando 'list' y el bucle 'for' de Python:

In [2]:
#Genero una lista con las alturas de la clase B
claseB = [172.5, 162.9, 161.7, 160.1, 161.8, 169.2]
alturaTotal = 0

for altura in claseB:
    alturaTotal += altura

print('Altura media clase B: ',alturaTotal/len(claseB))

Altura media clase B:  164.70000000000002


3). La clase A y la clase B se unen para formar la clase AB. Calcule la altura media utilizando 'list' y el bucle 'for' de Python:

In [3]:
#Uno la clase A y la clase B en una nueva lista
claseAB = claseA + claseB
alturaTotal = 0

for altura in claseAB:
    alturaTotal += altura

print('Media clases AB: ',alturaTotal/len(claseAB))

Media clases AB:  165.45000000000002


4). Ordenar las alturas de la clase AB.

In [4]:
claseAB.sort()
print(claseAB)

[160.1, 161.5, 161.7, 161.8, 162.9, 163.1, 165.9, 167.2, 168.7, 169.2, 170.8, 172.5]


5). La clase C se forma con estudiantes con una altura inferior a 165. Haga una lista que contenga las alturas de la clase C.

In [6]:
#Inicializo la variable claseC
claseC = []

for altura in claseAB:
    if altura < 165:
        claseC.append(altura)

claseC

[160.1, 161.5, 161.7, 161.8, 162.9, 163.1]

6). La clase D se forma con estudiantes con una altura mayor que 160 y menor que 170. Haga una lista que contenga las alturas de la clase D.

In [7]:
#Inicializo la variable claseD
claseD = []

for altura in claseAB:
    if altura > 165 and altura < 170:
        claseD.append(altura)
        
claseD

[165.9, 167.2, 168.7, 169.2]